### ISSUE TITLE
Issue with the SMPL-X face

### ISSUE THREAD

**Issue by RR-28023** [mks0601/ExAvatar_RELEASE#39]
Hi,

I'm trying to replicate your results with your own data and I've run into the following issue: the SMPL-x MODEL seems to not be renderign properly the face vertices. 

For example, below is my SMPL-X render:

![image](https://github.com/user-attachments/assets/c35148f2-58c0-4921-881e-b95725bfd34b)

And this is the one published in the project page:

![image](https://github.com/user-attachments/assets/6a7652a5-67ba-4016-b12d-74767536136b)

The face is cleary off in the one I'm getting. Any clue on what I may be doing wrong? I'm using FLAME 2020 as instructed.
Thanks!

**Comment by mks0601** [comment]
There are a couple of possible reasons, which is already reported in here: https://github.com/mks0601/ExAvatar_RELEASE/issues/13

1. `pip install smplx`2
2. Check DECA's results, which is available in $ROOT/flame_init/renders

And try the fitting again.

**Comment by RR-28023** [comment]
Thanks for the prompt response. Indeed it was an issue with the DECA step. I had not downloaded the `deca_model` checkpoint, and the process had not raised any error.

I've now succesfully trained the avatar with your example dataset. May I ask if the results shown or the github project page are with only 5 epochs of training (which is the default in the code)?, I ask because I get pretty decent results but not as good as yours. Thanks

**Comment by mks0601** [comment]
thanks for your confirmation. I added a line to ensure the availability of `deca_model.tar`: [line](https://github.com/mks0601/ExAvatar_RELEASE/blob/90335954adbbcfc3f149e477ca8586dcc28fd3d9/fitting/tools/code_to_copy/run_deca.py#L22).

I used only 5 epochs for all videos in the project page. Actually, you can download the avatar from [here](https://github.com/mks0601/ExAvatar_RELEASE/tree/main/avatar#pre-trained-checkpoints)

**Comment by RR-28023** [comment]
Understood. Then for some reason I can`t think off the avatar I get is slightly worse. See the picture below and compare it with the one above. Some details (e.g. the shoes black areas) are missing. Any suggestions on what I should review?
![image](https://github.com/user-attachments/assets/7cdb8996-f359-4c42-aaf7-e3ac5ddf1cf5)

**Comment by mks0601** [comment]
How did you make the avatar? Did you do all the fitting process by yourself? Or used provided pre-processed fitting data and just made the avatar based on the provided ones?

**Comment by RR-28023** [comment]
For the avatar I did everything by myself except for the segmentation. That is, the only initial data I had was the video recording and the masks.

For the SMPL-X parameters of the target video (`tvxq_mirotic`) I used the data provided by you.

**Comment by mks0601** [comment]
could you first check the quality of several dumped videos from the phone scan (not the target video)? you can compare the video with mine. for example, keypoints_whole_body.mp4 and smplx_optimized_smoothed.mp4

**Comment by RR-28023** [comment]
Yep okay that makes sense. I will see if I can resolve any discrepancies between my intermediate outputs and yours, and may open another issue if I can't solve it. Feel free to close this one, and thank you!
